pages/osx/lipo.md

```diff
@@ -0,0 +1,19 @@
+# lipo
+
+> Tool for handling Mach-O Universal Binaries.
```

Comment:
I don't think this describes the command very well. Especially, since it mentions Mach-0, which isn't mentioned again. What do you think about:
```suggestion
> Convert a universal binary to a single architecture file, or vice versa.
```

Reply:
I don't think that describes it well, because it doesn't only convert universal ↔ single-arch, it also converts universal ↔ universal, displays information about and "verifies" universal binaries. I think mentioning Mach-O in this way is exactly the right way, because `lipo` only operates on Mach-O, but mentioning that in the description of every example doesn't make sense to me. (it's repetitive)

Reply:
But what is Mach-0?

Reply:
[Mach-O, short for Mach object file format, is a file format for executables, object code, shared libraries, dynamically-loaded code, and core dumps.](https://en.wikipedia.org/wiki/Mach-O) It is Darwin's equivalent to ELF.